src/app/debug/table/table.component.ts

```diff
@@ -411,7 +438,7 @@ export class TableComponent implements OnInit, OnDestroy {
 
   disableReportInProgressButton(index: number, selector: string): void {
     let element: HTMLButtonElement = document.querySelector(selector)!;
```

Comment:
Please refactor this to the angular way.

Reply:
removed it as it was unnecessary. ngIf statement was made in an element before in html file so it would have disabled the element anyway